docs/faq.rst

```diff
@@ -711,25 +711,18 @@ then back it up:
 ::
 
     btrfs subvolume snapshot -r /home /.snapshots/home
-    borg create /mnt/borg::home123 /.snapshots/home
+    borg create /path/to/repo::home_backup /.snapshots/home
     btrfs subvolume delete /.snapshots/home
 
 The problem with this approach is that in a restore instead of paths starting with /home I
 have now /.snapshots/home.
 
-Solution 1: Run ``borg_create`` from within the snapshot directory.
+Solution: Run ``borg_create`` from within the snapshot directory.
 
 ::
 
     cd /.snapshots
-    borg create /mnt/borg::home123 home
-
-Solution 2: Use partial ``extract`` / ``mount`` by specifying a PATH.
-
-::
-
-    borg extract /mnt/borg::home123 /.snapshots/home
-    borg mount /mnt/borg::home123 /mnt/restore /.snapshots/home
+    borg create /path/to/repo::home_backup
```

Comment:
I just noticed that this is the solution to a way more general problem.

It is not just when snapshotting a btrfs subvolume (which mounts the snapshot), but basically when mounting any filesystem that is usually mounted elsewhere in the directory hierarchy. That could be also a bind-mount or a mount of a remote filesystem.

Also, I don't think it is needed to first show the commands for the wrong way, then point to the problem and then show the commands for the right way - just state the general problem and the solution.

Reply:
Made it so!